roles/aodh/defaults/main.yml

```diff
@@ -18,7 +18,7 @@ aodh:
   source:
     rev: 'stable/newton'
     constrain: True
-    upper_constraints: 'https://raw.githubusercontent.com/openstack/requirements/stable/newton/upper-constraints.txt'
+    local_constain: false
```

Comment:
Get rid of `local_constrain` variable. In openstack-source and clients role adjust logic such that, it relies on `not defined upper_constraints` to use local_contraints file. It is not at project level but more at which contraints file to use.